### DISCUSSION TITLE
Node Centrality Issue

### DISCUSSION THREAD

**Discussion by yikang7720** [pyg-team/pytorch_geometric#5146] [Q&A]
After I set the eigenvector centrality (by networkx package) as the node feature, it always run this error. 
KeyError: ('edge_index', tensor([[ 0,  0,  0,  ..., 89, 89, 89],
        [ 1,  2,  3,  ..., 84, 85, 87]]))
However, this error does not happen when I am using the node strength or node degree as the node feature.

**Comment by Padarn** [comment]
Can you provide a complete code example of the problem you are facing?
